client/components/Cart.js

```diff
@@ -0,0 +1,61 @@
+import React, {Component} from 'react'
+import { connect } from 'react-redux'
+import { fetchCart } from '../redux/cart'
+//all buttons need functionality: Checkout / keep shopping, remove from cart, + / - buttons 
+class Cart extends Component{
+    constructor(){
+        super()
+    }
+    componentDidMount(){
+        this.props.fetchCart()
+    }
+    calcTotal(costumes){
+        let cartTotal = 0
+        for(let costume of costumes){
+            cartTotal += costume.price
+        }
+        return cartTotal
+    }
+    render(){
+        const cart = this.props.cart
+        console.log(cart)
+        const costumes = cart.costumes ? cart.costumes : []
```

Comment:
it looks like defining the costumes variable in the render function works, but should this live in the component's state, defined in the constructor function?